library-best-practices/glossary.md

```diff
@@ -66,3 +72,61 @@ Java Dependency Glossary
 - **Linkage-matchable version** (said of a particular version of A in relation
   to all versions of B): There exists some version of B such that the version of
   A and the version of B are linkage-compatible.
+
+
+### Static Linkage Checking
+
+- **Static linkage**: a reference from a class to another class that is
```

Comment:
I think we should just treat "static" as a modifier for other terms, and not define "static linkage" at all.